cves/CVE-2013-1664.yml

```diff
@@ -1,296 +1,314 @@
+  note:
+vccs:
+- commit: 4ea7a11659b8a0ab07b0d2e847975f7324664f10
+  note: This VCC was discovered automatically via archeogit.
+- commit: c9032ab07f3694f3ae7da9b0017b764248ce28c9
+  note: This VCC was discovered automatically via archeogit.
+- commit: 953badbea5a04159adbfa970f5805c0232b6a401
+  note: This VCC was discovered automatically via archeogit.
+- commit: ff60c5f9de3e8690d1e86f3e9e3f7248a15397c8
+  note: This VCC was discovered automatically via archeogit.
+- commit: ea07351799ea71084c1279ad9f5dab1f81362c4b
+  note: This VCC was discovered automatically via archeogit.
+- commit: eedf16d12b675c50d0b0bab60ba8ad2e30a9921e
+  note: This VCC was discovered automatically via archeogit.
+- commit: 7ccf9978adab5c0846b13f804f1bab25436e2564
+  note: This VCC was discovered automatically via archeogit.
+- commit: ac64e91a0cadc57f4bc5cd5d66955832320ca7a1
+  note: This VCC was discovered automatically via archeogit.
+- commit: a33fb695e37c921d2bc0ffe3b011ad830f4bea56
+  note: This VCC was discovered automatically via archeogit.
+- commit: 7019123d21f202bfce54ad30eb33ae9f4b3bcef9
+  note: This VCC was discovered automatically via archeogit.
+upvotes_instructions: |
+  For the first round, ignore this upvotes number.
+
+  For the second round of reviewing, you will be giving a certain amount of
+  upvotes to each vulnerability you see. Your peers will tell you how
+  interesting they think this vulnerability is, and you'll add that to the
+  upvotes score on your branch.
+upvotes:
+unit_tested:
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were unit tests involved for this subsystem. The code
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  code: true
+  code_answer: true
+  fix: true
+  fix_answer: true
+discovered:
+  question: |
+    How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please explain where you looked.
+  answer: |
+    Was found by a Yahoo! employee when DOS was plaguing Nova keystone servers

```

Comment:
Could you leave a link or bug report here?

Reply:
Yea. Sure I was part of a discussion link on this issue